src/my_std.ml

```diff
@@ -25,6 +25,12 @@ exception Exit_system_error of string
 exception Exit_with_code of int
 exception Exit_silently_with_code of int
 
+type log = { mutable dprintf : 'a. int -> ('a, Format.formatter, unit) format -> 'a }
+(* Here to break the circular dep *)
+let dprintf _lvl _fmt = failwith "My_std.log not initialized"
+let log = { dprintf = dprintf }
```

Comment:
I would define `dprintf` as a local variable of `log`, it is strange to add a broken `dprintf` function to the global scope of the module.

Reply:
done